benches/raw_insertions.rs

```diff
@@ -2,10 +2,10 @@ use criterion::{criterion_group, criterion_main, Criterion};
 use rand::Rng;
 use std::hint::black_box;
 use vex_prover::imt::order::Order;
-use vex_prover::imt::IndexedMerkleTree;
+use vex_prover::imt::SellIMT;
```

Comment:
The updated import statement directly uses a concrete type SellIMT instead of potentially importing the broader module that contains both buy and sell side IMTs. This creates a tight coupling to a specific implementation.
```suggestion
use vex_prover::imt::{side::Sell, IndexedMerkleTree};
```

Reply:
fixed this in [pr/12](https://github.com/aerius-labs/vex-prover/pull/12)